test/Renci.SshNet.IntegrationTests/OldIntegrationTests/SshCommandTest.cs

```diff
@@ -51,6 +51,30 @@ public void Test_Execute_SingleCommand()
             }
         }
 
+        [TestMethod]
+        [Timeout(5000)]
+        public void Test_CancelAsync_Running_Command()
+        {
+            using var client = new SshClient(SshServerHostName, SshServerPort, User.UserName, User.Password);
+            #region Example SshCommand CancelAsync
+            client.Connect();
+            var testValue = Guid.NewGuid().ToString();
+            var command = $"sleep 10s; echo {testValue}";
+            using var cmd = client.CreateCommand(command);
+            try
+            {
+                var asyncResult = cmd.BeginExecute();
+                cmd.CancelAsync();
+                cmd.EndExecute(asyncResult);
+            }
+            catch (OperationCanceledException)
+            {
+            }
+            client.Disconnect();
+            Assert.AreNotEqual(cmd.Result.Trim(), testValue);
```

Comment:
```suggestion
            Assert.AreEqual("", cmd.Result);
```

? (if not, the arguments should be reversed at least)